addons/scoreboard/mergedplayer.lua

```diff
@@ -234,8 +234,21 @@ function MergedPlayer:wsavg()
     end
 end
 
-
-
+function MergedPlayer:wsacc()
+    local hits, misses = 0, 0
+    
+    for _, p in ipairs(self.players) do
+        for _ in pairs(p.ws) do hits = hits + 1 end
```

Comment:
This line is the same as:
```lua
        hits = hits + table.length(p.ws)
```

Only it avoids the explicit loop. (You probably took the code from the above function, which could also be simplified a bit.)